examples/machine-vsphere.yaml

```diff
@@ -25,7 +25,7 @@ spec:
     cloudProviderSpec:
       templateVMName: ubuntu-template
       username: '<< VSPHERE_USERNAME >>'
-      vsphereURL: 'https://<< VSPHERE_ADDRESS >>'
+      vsphereURL: '<< VSPHERE_ADDRESS >>'
```

Comment:
we could add a comment that would show an example value.

Reply:
I'll add an example